### ISSUE TITLE
Cannot upload the same folder in different subfolders

### ISSUE THREAD

**Issue by pionnegru** [Alfresco/alfresco-ng2-components#1732]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [x] Bug  
- [ ] Support request
- [ ] Documentation
```
**Current behaviour:**
Same folder cannot be uploaded in different subfolders. Uploading a folder in a subdirectory also creates that folder in the root.
**Expected behavior:**

**Steps to reproduce the issue:**

**Component name and version:** 

**Browser and version:** 

 
**Node version (for build issues):**  

**New feature request:**

![untitled-1](https://cloud.githubusercontent.com/assets/3947156/23955846/ca205714-09a3-11e7-9f6e-0cff1b87b2f6.gif)

**Comment by john-knowles** [comment]
Great to see this closed, will it be in the 1.3.0 milestone?

**Comment by DenisVuyka** [comment]
Yes, just updated the milestone.
